### DISCUSSION TITLE
Player Dosent work on WSA

### DISCUSSION THREAD

**Discussion by AmeyaGawde326** [aniyomiorg/aniyomi#1440] [General]
**How to Reproduce**
Start the app in WSA 
Play an episode
From any source

**Expected Behaviour**
The episode should start 

**Actual behaviour**
The application freezes crashes 

**Workarounds**
You can install an External Player like VLC and play the video through it and it works 

Which confirms my hypothesis that player itself is an issue rather than me running the app on WSA

**Comment by LittleFreak** [comment]
It's probably better to open this as an issue, instead as an discussion. =)
